README.md

```diff
@@ -1,22 +1,103 @@
+   If Git isn't already installed, running `git` from your terminal will often prompt to install it.
+
+3. **Install FMOD Engine**  
+   - Go to the FMOD download page: [FMOD v1.10.09](https://www.fmod.com/download?version=1.10.09#modengine).  
+   - You will need to create a free FMOD account to download.
+   - Make sure that the FMOD version is set to **v1.10.09 (unsupported)**
+   - Download the **iOS** version of **FMOD Engine**
+   - After downloading, mount the downloaded `fmodstudioapi11009ios-installer.dmg` file by double-clicking it. **No need to do anything further**; simply ensure it's mounted.
+
+4. **Install Visual Studio for macOS**  
+   - Download Visual Studio for macOS from [here](https://visualstudio.microsoft.com/vs/older-downloads/).
+     - If that download link doesn't work, use [this one](https://visual-studio.macupdate.com)
+   - Follow the websites links until it lets you download **Visual Studio 2022 for Mac**
+   - Follow the installation instructions.
+   - When prompted during installation, make sure to select the following components:
+     - `.NET`
+     - `.Net Multi-Platform App UI`
+     - `Android`
+     - `iOS`
+   - You do **not** need to sign into Visual Studio, but the installation will require your macOS password.
+   - Once the installation is complete and you see the option to create or open a project, simply close the program. We won’t need to open Visual Studio after this.
+
+## Installing Celeste (FNA Linux Build)
+
+### 1. **From itch.io**  
+   - You can download/buy Celeste directly [here](https://mattmakesgames.itch.io/celeste).  
+   - After purchase or login, download the **Celeste Linux** Version.
+
+### 2. **From Steam**  
+   - Make sure Steam is installed on your system.
+   - Make sure Steam is **not** open.
+   - Type `steam://open/console` into the URL bar of your browser, and allow it to open Steam.
+   - Steam should open with a new **Console** Tab, go into it.
+   - Type `download_depot 504230 504233 1505052356460012099` into the Steam console and wait for it to install (there won't be a progress bar).
+   - Once complete, it will tell you where the files were installed. It will say something like this:
+   
+     ```bash
+     Depot download complete: "/Users/username/Library/Application Support/Steam/Steam.AppBundle/Steam/Contents/MacOS\\steamapps\\content\\app_504230\\depot_504233" (1259 files, manifest 1505052356460012099)
+     ```
+
+   - In this case, `depot_504233` is your Celeste folder that you should give to the build script. *This may vary.*
+   - Please move this folder to somewhere else, like your Downloads, or Documents folder, you can also rename it something like `Celeste-Linux`
+
+Once you have installed Celeste, make sure the folder contains content such as `Celeste.exe`, `Content` (folder), `Backups` (folder), `lib` (folder), `lib64` (folder).
+
+Also, Make sure the Name of this folder has no spaces in it, and make sure the path it's in has no spaces in it.
```

Comment:
this is a bug - shouldn't be mentioned in the README, but rather on the issue tracker ^^ (see #8)

Reply:
No worries, I just put it in because I thought it was a mistake that most people probably could make, and because I doubt I could fix it, I thought it was best to put it in here just so people avoid making it, But yeah again I'm happy to remove it :)